bigquery/google/cloud/bigquery/client.py

```diff
@@ -341,6 +341,19 @@ def dataset(self, dataset_id, project=None):
 
         return DatasetReference(project, dataset_id)
 
+    def _create_bqstorage_client(self):
+        """Create a BigQuery Storage API client using this client's credentials.
+
+        Returns:
+            google.cloud.bigquery_storage_v1beta1.BigQueryStorageClient:
```

Comment:
Should the return type be cross-linked in the docs? I personally find that useful.
```suggestion
            :class:`google.cloud.bigquery_storage_v1beta1.BigQueryStorageClient`:
```

Reply:
This method is private, so it won't show up in the docs anyway.

For `Returns:` in general, an explicit `:class:` link isn't necessary. For example, below you can see the `create_dataset` method does not use an explicit `:class:` link, but the Return Type still renders as a link.

![image](https://user-images.githubusercontent.com/247555/68158136-ded50d00-ff03-11e9-9f7d-e70ea8002c66.png)

Reply:
I see, all good then.